### PR TITLE
Polymer 3 Conversion

### PR DESCRIPTION
**Pull request by AdmiralCoco**
Looping in @svanherk @awikkerink in case you'd like to review (can't add reviewers)

Also @CodeBaboon would you be able to create the `hybrid` branch on Brightspace's repo?

### CODE REVIEW

**Comment by CodeBaboon** [comment]
@AdmiralCoco would that just be a branch from the current `master` head commit?

File: package.json [hunk 1]
```diff
@@ -1,36 +1,52 @@
...
+    "type": "git",
+    "url": "https://github.com/Brightspace/d2l-fetch-siren-entity-behavior.git"
+  },
+  "homepage": "https://github.com/Brightspace/d2l-fetch-siren-entity-behavior",
+  "name": "d2l-fetch-siren-entity-behavior",
+  "scripts": {
+    "test": "npm run test:lint:js && npm run test:lint:wc && npm run test:unit:local",
+    "test:lint:js": "eslint . --ext .js,.html test/**/*.js test/**/*.html demo/**/*.js demo/**/*.html",
+    "test:lint:wc": "polymer lint",
+    "test:unit:local": "cross-env LAUNCHPAD_BROWSERS=chrome polymer test --skip-plugin sauce"
```
Comment (awikkerink) [inline package.json]: You might need to add `--npm` for the unit tests to pass

**Comment by CodeBaboon** [comment]
@AdmiralCoco I know very little of Polymer 3 at this point. It seems legit? :shrug:

**Comment by AdmiralCoco** [comment]
@awikkerink if you don't have any other comments, we can merge? 🙂

**Pull request merged**